examples/streaming/app.js

```diff
@@ -0,0 +1,32 @@
+var fs = require('fs'),
+    path = require('path'),
+    express = require('express'),
+    request = require('request'),
+    dust = require('dustjs-linkedin');
+
+dust.config.whitespace = true;
+
+dust.onLoad = function(tmpl, cb) {
+  fs.readFile(path.join('./views', tmpl + '.dust'), { encoding: 'utf8' }, cb);
```

Comment:
nit: this kinda scares me a little to allow a tmpl name to drive an fs.read without sanitizing it. I know this is all server side but the source is compiled and loadsourced ....

maybe escape/sanitize against `..` or pre-load the templates?